globoi/marketplace-apple
# Fix realtime progress to update progressbar on current programs

To test this:
- Open EPG and wait for a current program to change in a channel. The progressbar from the old program should disappear and the new program should have a progressbar shown with a progress of 0.
